wordpress-mobile/WordPress-iOS
# Jetpack Security Settings

**Implements** #7474 

**To test:**

On a Jetpack site, go to Settings->Jetpack->Security. Check that updating a setting in the app is reflected on the web for each of the following settings:

- [x] Monitor uptime 
- [x] Send notifications via email
- [x] Send push notifications
- [x] Block malicious login attempts
- [x] Whitelisted IP addresses
- [x] Allow WordPress.com login
- [x] Match accounts using email
- [x] Require two-step authentication
- [x] Learn more link
 
Note: There's no validation for the whitelisted IP addresses, I'm doing the same thing Calypso does, the user can enter whatever they want and upon save the list is sanitized. Since I don't expect this feature to receive heavy usage on mobile I don't think it's worth the effort to also have validation client side.

Needs review: @koke
